target/linux/ramips/image/Makefile

```diff
@@ -79,6 +79,18 @@ define Build/mkdlinkfw
 		-s $(DLINK_FIRMWARE_SIZE)
 endef
 
+define Build/mkdlinkfw-loader
+	dd if=$(IMAGE_KERNEL) >> $@.tmp
+	dd if=$(IMAGE_ROOTFS) >> $@.tmp
```

Comment:
Why not keep/use append-kernel/append-rootfs instead of this?

Reply:
Ah, now I see that this is motivated by how mkdlinkfw has been used previously. In that case, I don't have a preference.

Reply:
Your and @mpratt14 solution is better IMO.